manifests/init.pp

```diff
@@ -117,5 +70,4 @@
   class { '::ntp::config': } ~>
   class { '::ntp::service': } ->
   anchor { 'ntp::end': }
```

Comment:
In a pinch, you can always set `ntp::service_manage` to `false` to avoid this, and replicate the [service resource](https://github.com/puppetlabs/puppetlabs-ntp/blob/55a3166/manifests/service.pp). I know that is not a optimal solution, but the main focus of this patch set is **not** changing anything, as perverse as that may sound.